content/docs/start/experiments.md

```diff
@@ -18,17 +18,90 @@ once they're no longer needed.
 
 ## Running experiments
```

Comment:
This section seems too long. Can you separate setup into its own section?

Reply:
Are you referring to the stuff in the ` <details>` block @dberenbaum ? (Done per https://github.com/iterative/dvc.org/pull/2491#discussion_r637487171 I think)

Seems OK to have that expandable section there, but let's try to shorten it as much as possible @iesahin ? Here's how...

Reply:
I mean that `Running experiments` includes:
1. Setup
2. Running a baseline experiment
3. Showing the experiment results
4. Running an experiment with different param values

It's not that any individual topic is too long, but it might be too many concepts under one section. Can we break up into smaller sections?

Reply:
This is done in #2574

Reply:
The sections in #2574  are smaller.